clients/drcachesim/tests/scheduler_unit_tests.cpp

```diff
@@ -7471,6 +7471,175 @@ test_record_scheduler()
     check_next(stream0, record_scheduler_t::STATUS_EOF);
 }
 
+static void
+test_record_scheduler_i7574()
+{
+    // Demonstrates how the scheduler responds to traces with the i#7574 issue.
```

Comment:
The checks below are actually checking for the buggy behavior, right?  How about another TODO up here so it's clear this test is not actually testing proper behavior.